TwitchDownloaderCLI/Modes/Arguments/ChatDownloadArgs.cs

```diff
@@ -0,0 +1,43 @@
+﻿using CommandLine;
+using TwitchDownloaderCore.Options;
+
+namespace TwitchDownloaderCLI.Modes.Arguments
+{
+
+    [Verb("chatdownload", HelpText = "Downloads the chat from a VOD or clip")]
+    public class ChatDownloadArgs
+    {
+        [Option('u', "id", Required = true, HelpText = "The ID of the VOD or clip to download that chat of.")]
+        public string Id { get; set; }
+
+        [Option('o', "output", Required = true, HelpText = "Path to output file.")]
+        public string OutputFile { get; set; }
+
+        [Option('b', "beginning", HelpText = "Time in seconds to crop beginning.")]
+        public int CropBeginningTime { get; set; }
+
+        [Option('e', "ending", HelpText = "Time in seconds to crop ending.")]
+        public int CropEndingTime { get; set; }
+        
+        [Option('E', "embed-emotes", Default = false, HelpText = "Embed emotes into the chat download.")]
```

Comment:
In my opinion, since people interfacing with the CLI will need to rework their code to update this (it is breaking e.g. not back compatible), you might as well make all the args consistent in my opinion (e.g., lowercase for all short commands).

This could be `-d` and `--download-emotes` and `Download and embed emotes into the chat download file` for example.

Reply:
I just now see the `ConvertFromOldSyntax` maybe things are ok as is, you already have kept backwards compatible.

Reply:
Yes the rework is fully backwards compatible!

I figured since embed emotes is so common I may as well give it a short char, and `E` seemed good to me since both words start with e's